config/puma/production.rb

```diff
@@ -13,20 +13,17 @@
 #
 workers ENV.fetch('FOREMAN_PUMA_WORKERS', 2).to_i
 
-# Provides systemd notify support through the
-# puma-systemd-plugin
-begin
-  plugin :systemd
-rescue Puma::UnknownPlugin
-  puts "Failed to load systemd plugin"
-end
-
 # In clustered mode, Puma can "preload" your application. This loads all the
 # application code prior to forking. Preloading reduces total memory usage of
 # your application via an operating system feature called copy-on-write
 #
 preload_app!
 
+# When systemd socket activation is detected, only use those sockets. This
+# makes FOREMAN_BIND redundant. The code is still there for non-systemd
+# deployments.
+bind_to_activated_sockets 'only'
+
```

Comment:
Can you also drop a comment next to `FOREMAN_BIND` use in this file for the future reference (just to explain it's no longer needed).

Reply:
It is needed on non-systemd activated socket deployments. Containers are one example.